<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTPlayerClass</class>
 <widget class="QWidget" name="QTPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#openButton:!hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(120, 120, 120, 255),stop:0.495      rgba(39, 39, 39, 255),stop:0.505 rgba(39, 39, 39, 255),stop:1 rgba(120, 120, 120, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
#openButton:hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(39, 39, 39, 255),stop:0.495      rgba(120, 120, 120, 255),stop:0.505 rgba(120, 120, 120, 255),stop:1 rgba(39, 39, 39, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
#playButton{background-color: rgba(255, 255, 255,0);}

/*#playButton:hover{image: url(:/ffplay/PlayN.png);}

#playButton:!hover{image: url(:/ffplay/PlayH.png);}

#playButton:pressed{image: url(:/ffplay/PlayP.png);}
*/
 

#bottemWidget{
background-color: rgba(0, 0, 0, 122);
}

#totalHour{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#totalMinute{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#totalSecond{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentHour{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentMinute{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentSecond{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="VideoOpenGLWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="bottemWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>509</y>
     <width>800</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="0,0,0,0,1,2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="3">
     <widget class="QWidget" name="totalContainer" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="totalHour">
         <property name="text">
          <string>0:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalMinute">
         <property name="text">
          <string>0:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalSecond">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="currentContainer" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="currentHour">
         <property name="text">
          <string>0:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentMinute">
         <property name="text">
          <string>0:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentSecond">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QSlider" name="playslider">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="playButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="openButton">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QSlider" name="volumeSlider">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videoopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/PlayRes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>openVideoFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderPressed()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderReleased()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>setVolume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openVideoFile()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>setPos(int)</slot>
  <slot>play()</slot>
  <slot>setVolume(int)</slot>
  <slot>mirrorLeftAndRight()</slot>
  <slot>mirrorUpAndDown()</slot>
 </slots>
</ui>
